ui/app/AppLayouts/Wallet/HistoryTab.qml

```diff
@@ -5,14 +5,29 @@ import "../../../imports"
 import "../../../shared"
 
 Item {
-    function checkIfHistoryIsBeingFetched(){
-      if(walletModel.isFetchingHistory(walletModel.currentAccount.address)){
-        loadingImg.active = true;
-      } else {
-        walletModel.loadTransactionsForAccount(walletModel.currentAccount.address)
-      }
+    property var tokens: {
+        const count = walletModel.defaultTokenList.rowCount()
+        const toks = []
+        for (var i = 0; i < count; i++) {
+            toks.push({
+                          "address": walletModel.defaultTokenList.rowData(i, 'address'),
+                          "symbol": walletModel.defaultTokenList.rowData(i, 'symbol')
+                      })
+        }
+        return toks
     }
```

Comment:
Sorry, I reformatted the file since it had 2 spaces.
This is one of the places where I actually did changes